apps/web/components/ui/form/Select.tsx

```diff
@@ -110,6 +110,12 @@ function Select<
             color: "var(--brand-text-color)",
           },
         }),
+        dropdownIndicator: (styles) => {
```

Comment:
@sean-brydon Right now there are many select, so I guess it would go everywhere.

Also, @gitstart-calcom can you confirm this change is supposed to apply to all selects as per the design.

Reply:
Removed the styling to keep it consistent with other dropdown styles